code/src/css/MovieDetails.css

```diff
@@ -0,0 +1,75 @@
+.movie-details {
+    display: flex;
+    align-items: flex-end;
+    height: 100vh;
+    color: white;
+    padding: 50px;
+    align-self: flex-start;
+}
+
+.background {
+    box-sizing: border-box;
+    z-index: -1;
+    min-height: 100vh;
+    min-width: 100%;
+    background-size: cover;
+    position: absolute;
+    top: 0;
+    left: 0;
+}
+
+.overlay {
+    background: linear-gradient(180deg, rgba(240, 238, 238, 0) 0%, rgba(31, 31, 31, 0) 86%, rgba(0, 0, 0, 1) 100%, rgba(0, 0, 0, 1) 100%);
+    width: 100%;
+    height: 100%;
+    content: '';
+    display: inline-block;
+    position: absolute;
+    left: 0;
+    top: 0;
+    overflow: hidden;
+}
+
+.back-button {
+    color: white;
+    position: absolute;
+    display: inline-flex;
+    align-items: center;
+    left: 50px;
+    top: 50px;
+    font-weight: 500;
+}
+
+.back-button svg {
+    width: 30px;
+    transition: 0.3s ease;
+}
+
+.back-button:hover svg {
+    margin-right: 5px;
+}
+
+.poster {
+    display: flex;
+    border: 4px solid white;
+}
+
+.movie-info {
+    display: flex;
+    align-items: flex-end;
+    align-self: flex-end;
+    z-index: 1;
+}
+
+.details {
+    font-weight: 300;
+    margin-left: 25px;
+    max-width: 400px;
+}
+
+.vote {
+    font-weight: 300;
+    font-size: 17px;
+    color: red;
+    margin-left: 10px;
```

Comment:
Great work with style! Do not forget to make it responsive if you send the case to a recruiter. You never know which device they are using. :)